arangod/Aql/ExecutionNode.h

```diff
@@ -612,7 +632,7 @@ class SingletonNode : public ExecutionNode {
 
     cloneHelper(c, withDependencies, withProperties);
 
-    return static_cast<ExecutionNode*>(c);
```

Comment:
does this need to be cast at all ?

Reply:
No, it doesn't. I just did a global search & replace.
I will tidy up that place later.